### ISSUE TITLE
v1.50.0 Fails gem install (self-runner)

### ISSUE THREAD

**Issue by jdaviderb** [ruby/setup-ruby#106]
# Problem
The most recent release brokes our build

![image](https://user-images.githubusercontent.com/4649902/98703077-1eb9a600-2349-11eb-97e2-91110d1150a2.png)

**self-runner: Ubuntu 18.04.5 LTS**

```bash
ubuntu:~/actions-runner$ lsb_release -a
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.5 LTS
Release:	18.04
Codename:	bionic
```

```bash
echo $RUNNER_TOOL_CACHE
/opt/hostedtoolcache
```

# Notes

when I rollback to `1.49.0` the build passed

![image](https://user-images.githubusercontent.com/4649902/98703184-3c870b00-2349-11eb-876f-e3ab2b256dcd.png)

**Comment by eregon** [comment]
Seems a duplicate of https://github.com/ruby/setup-ruby/issues/104.
It looks like `$RUNNER_TOOL_CACHE` as read from the action was `/home/runner/_work/_tool` and not `/opt/hostedtoolcache`.

https://github.com/ruby/setup-ruby/issues/104#issuecomment-724618159 might help.

**Comment by eregon** [comment]
Fixed by ba761ba8ff71a9113d3207595ceadac55aa09cef

**Comment by jdaviderb** [comment]
thanks @eregon now it's working fine 💯 🥇
